charts/everest/templates/everest-operator/backupstorage-editor.clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: everest-operator-backupstorage-editor-role
+rules:
+- apiGroups:
+  - everest.percona.com
+  resources:
+  - backupstorages
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - everest.percona.com
+  resources:
+  - backupstorages/status
+  verbs:
+  - get
```

Comment:
Don't we need a ClusterRoleBinding to bind all these ClusterRoles to the ServiceAccount?

Reply:
I was actually thinking if should have added these ClusterRoles at all because no binding was generated so I think that they are created thinking about some future service account but aren't directly used by the operator. Do you think we should remove them since they are not being used?

Reply:
Okay.. If they serve no purpose, I guess lets just remove them 😛

I also don't see any harm in keeping them..